openhands/agenthub/codeact_agent/tools/search_engine.py

```diff
@@ -0,0 +1,25 @@
+from litellm import ChatCompletionToolParam, ChatCompletionToolParamFunctionChunk
+
+_SEARCH_ENGINE_DESCRIPTION = """Execute a web search query (similar to Google search).
+
+You MUST use this tool as a search engine and find URLs of webpages that are relevant to the search query.
```

Comment:
```suggestion
Use this tool when you need a search engine to find URLs of webpages that are relevant to the search query.
```

Maybe? Just wondering if telling the agent "MUST" might make it use it more than necessary, to the detriment of `web_read` or other actions.

Reply:
Changed the description to be less prescriptive and more suggestive.